docker-compose.yml

```diff
@@ -1,40 +1,47 @@
 version: '3'
 services:
+  azurite:
```

Comment:
@chriscoffee should we configure the unit tests to talk to azurite?

Also https://github.com/gomods/athens/pull/973 could use this as well 😄